app/models/hike.rb

```diff
@@ -1,3 +1,9 @@
 class Hike < ApplicationRecord
   belongs_to :user
+  after_create :set_defaults
+
+  def set_defaults
+    self.image ||= "https://i.pinimg.com/originals/77/85/91/7785910e63b1662e8abe313c8ef9d160.jpg"
```

Comment:
This will set the image into the Hike model, AFTER it is created.  But it does not save the image there.  Perhaps this could be a `before_create` ?